src/data/git-quiz.ts

```diff
@@ -486,7 +486,7 @@ const gitQuiz = [
   },
   {
     Question: "In Git, how do you see a summary for a single stash?",
-    Answer: "git stash show",
+    Answer: "Git stash show",
```

Comment:
this change is extraneous for this PR, please remove it

Reply:
Removed